prov/verbs/src/fi_verbs.h

```diff
@@ -249,6 +249,8 @@ struct fi_ibv_eq {
 	fastlock_t		lock;
 	struct dlistfd_head	list_head;
 	struct rdma_event_channel *channel;
+	struct dlist_entry domain_list;
+	fastlock_t domain_list_lock;
```

Comment:
nit: please align new fields with other fields in this structure.

Is a new lock needed?  We already have a lock available.